cpanfile

```diff
@@ -0,0 +1,31 @@
+use strict;
+use warnings;
+
+on 'runtime' => sub {
+    requires 'Pod::Text' => '4.00';    # to avoid issues with wrong test results
+    requires 'Cwd';
+    requires 'File::Basename';
+    requires 'File::Spec' => '0.82';
+};
+
+on 'configure' => sub {
+    requires 'ExtUtils::MakeMaker';
+};
+
+on 'test' => sub {
+    requires 'Test::More' => '0.60';
+    requires 'blib';
+};
+
+on 'develop' => sub {
+    requires 'Dist::Zilla';
```

Comment:
I don't generally recommend including dzil requirements in develop-requires (dzil itself or the plugins/pluginbundles) since dzil can manage them itself using `dzil authordeps`. The rest of these develop requires should get automatically added by the associated plugin so shouldn't be necessary either (but double check that this actually happens).

Reply:
I removed the Dist::Zilla stuff from develop via ba08de53
I'm leaving the other develop entries as this is enforcing some versions
I could have also probably move all that cpanfile to the dist.ini itself... using the appropriate sections...

Reply:
I prefer https://metacpan.org/pod/Dist::Zilla::Plugin::PrereqsFile + prereqs.yml for specifying dzil prereqs lately but it's up to you.

Reply:
I agree PrereqsFile format looks nicer to read but the cpanfile can easily be plugged in alternate locations, for example to install prereqs for CI

Reply:
The Makefile.PL and META.json are regenerated so CI should be able to use that to install prereqs regardless of whether it uses dzil.